buildSrc/src/main/resources/com/uber/okbuck/core/util/retrolambda/RetroLambda.sh

```diff
@@ -1,4 +1,3 @@
 #!/bin/bash
 
-DEPS=$(JARS=(`find gen-dir ! -name '*-abi.jar' ! -name '*dex.dex.jar' -name '*.jar'`); IFS=:; echo "${JARS[*]}")
-java -Dretrolambda.inputDir=$1 -Dretrolambda.classpath=$1:android-jar:$DEPS -jar retrolambda-jar
+java -Dretrolambda.inputDir=$1 -Dretrolambda.classpath=$1:$ANDROID_JAR:$DEPS -jar retrolambda-jar
```

Comment:
Can we treat the `$ANDROID_JAR` as a part of the `$DEPS`? Will make the logic simpler as there is no real need to distinguish between the two really